Documentation/Signoffs/SensorSignoff.md

```diff
@@ -0,0 +1,94 @@
+# Sensor Subsystem Signoff
+## FUNCTION OF THE SUBSYSTEM
```

Comment:
The first paragraph is not clear. What commands are taken? What tasks are executed? What does "analyze and receive data" mean? Also the last two sentences don't contribute to explaining the functionality.

Second paragraph - rewrite in terms of functionality...what needs to be done?

Reply:
Going off of the pseudo code within the Microcontroller Signoff, the tasks are to start collecting the data, and after calculation, send the data to the interface and back to the microcontroller. Data acquisition is the receiving and analyzing of data. The 2nd to last sentence you are referring to in the fist paragraph creates a segway to the next paragraph in the Function analysis. I explain in further detail how each of these sensors calculate the outside parameter. I will delete the last sentence.

Well, the subsystem must be modular. So I explained in detail how each of the different type of sensors calibrate their data to send. I believe the function of the subsystem as a whole has been explained.

Not sure what you are looking for? I will mention to sampling rate later in the document, but am having difficulty finding a social/global concern for the sensors being used in the project.